src/Form/Field/BlocksField.php

```diff
@@ -222,10 +222,6 @@ public function validations(): array
                     $index++;
                     $blockType = $block['type'];
 
-                    if (isset($fields[$blockType]) === false) {
-                        continue;
-                    }
-
```

Comment:
This lines was preventing the bottom lines from running. Tell anyone who knows what it does 😅

Reply:
It skips validation for blocks with unknown block types. This will make sure that the data for such unknown blocks will still be stored. Otherwise migrated blocks from builder and editor would create a validation error and also simply be removed after saving the blocks field for the first time.

Reply:
Are we sure? `$fields` setted empty array. It will always be empty.

Reply:
Yikes. You are right. The check needs to be further down below after fetching the fields. 🙈

Reply:
I have updated for layout and blocks validation. Used try-catch due to `fields()` method throw exception. Is it right now?